conf/messages.en

```diff
@@ -255,6 +257,18 @@ addATrustee.individual = Trustee Individual
 addATrustee.business = Trustee Business
 addATrustee.error.required = Select yes if you want to add another trustee
 
+trusteeAUKCitizen.title = Is {0} a UK citizen?
+trusteeAUKCitizen.heading = Is {0} a UK citizen?
+trusteeAUKCitizen.checkYourAnswersLabel = trusteeAUKCitizen
```

Comment:
You need to implement the correct message for this key.

Reply:
fixed